projects/ngrx-helpers/src/store/ngrx-effect.ts

```diff
@@ -15,7 +15,7 @@ export abstract class NgrxEffect {
 
   public effect(effectConfig: NgrxEffectConfig): Observable<any> {
     return this.action$
-      .ofType(effectConfig.action) // tslint:disable-line
+      .pipe(ofType(effectConfig.action))
       .pipe(
```

Comment:
instead of new pipe, lets include in the same pipe and as first input

```suggestion
      .pipe(
        ofType(effectConfig.action),
```

Reply:
Fixed.

Reply:
published 1.1.1